src/components/User/User.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const User = ({ user }) => {
+  const { name, email, address } = user;
+
+  return (
+    <>
+      <p>{name}</p>
+      <p>{email}</p>
+      <p>
+        {address.suite}
+        ,
+        {' '}
+        {address.street}
+        ,
+        {' '}
+        {address.city}
+        ,
+        {' '}
+        {address.zipcode}
```

Comment:
write this in one line like this {`your text goes here and here's a ${variable}`}